Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Source.jinja

```diff
@@ -318,7 +318,11 @@ void {{ ClassName }}::{{ UpperFirst(Property.attrib['Name']) }}({{ ', '.join(par
     constexpr AzNetworking::ReliabilityType isReliable = Multiplayer::ReliabilityType::Unreliable;
 {% endif %}
 
+{%     if InvokeFrom == 'Server' and HandleOn == 'Authority' %}
```

Comment:
I didn't want to add an extra flag here since all the relevant information should already be captured. That being said, I'm not 100% sure if this covers all cases or if I'm missing some.

Reply:
I think you want InvokeFrom == 'Server' or InvokeFrom == 'Client', ... That captures extra cases that are not really supported (we don't allow a client to issue an RPC), but it also precisely targets the cases where an invocation would happen on a component and not have a GetParent() method, making it a slightly more defensive change in case someone ever decides to add Client to Autonomous or Client to Authority RPCs.

Reply:
Cool, this is exactly what I was unsure of. I'll make that change.